clang/lib/Sema/SemaExpr.cpp

```diff
@@ -11428,7 +11428,7 @@ static bool isScopedEnumerationType(QualType T) {
   return false;
 }
 
-static void DiagnoseBadShiftValues(Sema& S, ExprResult &LHS, ExprResult &RHS,
+static void DiagnoseBadShiftValues(Sema &S, ExprResult &LHS, ExprResult &RHS,
```

Comment:
```suggestion
static void DiagnoseBadShiftValues(Sema& S, ExprResult &LHS, ExprResult &RHS,
```
Backing out an unrelated formatting change.